WebKitBrowser/WebKitImplementation.cpp

```diff
@@ -2267,6 +2429,17 @@ static GSourceFuncs _handlerIntervention =
 
             URL(static_cast<const string>(_URL));
 
+            _webProcessState = WebProcessCold;
+
+            if(webkit_web_view_get_is_web_process_responsive(_view) && _webProcessState == WebProcessCold)
+            {
+                _webProcessState = WebProcessHot;
+            }
+
```

Comment:
This is not reliable to transition to a hot state. lets discuss how to make it differently.

Reply:
Changed the place to onURLChanged.